tools/python/CMakeLists.txt

```diff
@@ -13,35 +13,10 @@ endif()
 
 project(dlib_python_bindings)
 
-# Pybind11's cmake scripts enable link time optimization by default.  However,
-# it makes linking take a really long time and doesn't seem to substantively
-# improve runtime performance.  So we disable LTO here to make building dlib
-# faster.
-set(PYBIND11_LTO_CXX_FLAGS "")
-
-
-# Avoid cmake warnings about changes in behavior of some Mac OS X path 
-# variable we don't care about.
-if (POLICY CMP0042)
-    cmake_policy(SET CMP0042 NEW)
-endif()
-
-
-# To avoid dll hell, always link everything statically when compiling in
-# visual studio.  This way, the resulting library won't depend on a bunch
-# of other dll files and can be safely copied to someone else's computer
-# and expected to run.
-if (MSVC)
-    include(${CMAKE_CURRENT_LIST_DIR}/../../dlib/cmake_utils/tell_visual_studio_to_use_static_runtime.cmake)
-endif()
```

Comment:
This is totally needed.  Not having this will blow people's minds since the notion of "you must have dependent library dll files in your PATH" is not a concept that is at all familiar to the windows python community.